spark-jobs/src/main/scala/filodb/downsampler/chunk/BatchExporter.scala

```diff
@@ -65,41 +54,22 @@ case class BatchExporter(downsamplerSettings: DownsamplerSettings, userStartTime
   val exportSchema = {
     // NOTE: ArrayBuffers are sometimes used instead of Seq's because otherwise
     //   ArrayIndexOutOfBoundsExceptions occur when Spark exports a batch.
-    val fields = new mutable.ArrayBuffer[StructField](3 + downsamplerSettings.exportPathSpecPairs.size)
+    val fields = new mutable.ArrayBuffer[StructField](11)
     fields.append(
-      StructField("LABELS", StringType),
-      StructField("TIMESTAMP", LongType),
-      StructField("VALUE", DoubleType)
+      StructField("workspace", StringType),
```

Comment:
Could we add some config to specify labels to include in the fields? Something like:
```
label_fields = [
   "l1", "label1",
   "l2", "label2"
]
```

Reply:
As the schema is fixed containing some additional label based columns, in the latest commit, I have added a new constants files for schema. This file defines which keys from labels map will be used to populate additional label based columns in the table.

Reply:
Implemented the dynamic schema generation concept for Iceberg Table as per the additional cols specified in conf file as above.